redcap_bridge/test_redcap/test_project_validation.py

```diff
@@ -0,0 +1,47 @@
+import pathlib
+from redcap_bridge.redcap_validator import load_template, load_records, validate
+
+
+def test_load_template():
```

Comment:
Maybe just add a little description of what we are testing and how it works  WDYT ?

Reply:
This function was removed during refactoring.